src/playground-ui/VisNetworkGraph.tsx

```diff
@@ -58,6 +58,15 @@ interface VisGraphDefinition<N extends VisNode, E extends VisEdge> {
     nodes: N[], edges: E[]
 }
 
+interface VisDataSet {
+    update: (data: object) => void
```

Comment:
`object` is like one step above `any`, but it's useful when you just want to say "yeah it's an object"